### ISSUE TITLE
Increase the baseline for claiming accessible epubs

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2717]
### Section

https://www.w3.org/TR/epub-a11y-111/#sec-wcag-conf

### Describe the problem

It has been requested that we increase the floor for conformance from WCAG 2.0 Level A to 2.1 or 2.2 at Level AA.

We currently only recommend that publishers meet the current version of WCAG at Level AA because EPUB is a global standard and accessibility laws are not uniform.

Moreover, all of the WCAG 2.X line of documents are W3C Recommendations and the latest ones do no supersede the earlier ones. This is also because legislation differs and not everyone has to meet the same standards.

Both standards are in the same boat here, and it's not clear that we should be moving to increase the requirements on publishers before W3C obsoletes the older WCAG versions. Where accessibility is required by law, we also don't have to be concerned about allowing conformance to older versions and levels because, as already stated in the specification, local law always takes precedence. The lower level bar can be seen as an incentive to begin meeting international norms where accessibility is not yet mandated.

In any case, before we seek to change the conformance requirements, we need more input on how this will affect producers. Please feel free to provide comments both for and against changing the current requirements below.

### Describe the fix or new feature you propose

A smaller change might be to increase the WCAG conformance level without changing the version requirement. Most legislation requires level AA conformance, so the only impact would be on publishers who are not mandated to meet accessibility requirements, but this may not be a significant number of producers making accessible EPUB content. It just takes away the minimal level of conformance.

But this would still be a significant change from a specification conformance perspective, so it would require bumping the version number to 1.2.

**Comment by iherman** [comment]
I am, I admit, out of my depth here. But, unless I am mistaken, this means there might be publications that are conforming per version 1.1 of the spec that would not be conforming per 1.1.1 (or 1.2). If that is true, wouldn't that go against the [current charter](https://www.w3.org/2025/02/pmwg-charter.html#scope)? 

(Although the charter mentions only EPUB 3.3 vs. 3.4 in its backward compatibility clause, I think it was also meant for the A11y specs.)

> But this would still be a significant change from a specification conformance perspective, so it would require bumping the version number to 1.2.

If I remember correctly, there were long discussions among a11y experts in this group on the choice between 1.2 and 1.1.1, with a final decision made for the latter. I do not think it would be a good sign to change this just a few weeks after republication...

**Comment by mattgarrish** [comment]
> Although the charter mentions only EPUB 3.3 vs. 3.4 in its backward compatibility clause, I think it was also meant for the A11y specs.

I'm not sure how realistic it is to expect accessibility to stay rooted in time in the past. It's an issue we need to consider, but conformance usually works the other way around - what you produce now should be compatible with what came before, not what you produced in the past remains conforming today.

> I do not think it would be a good sign to change this just a few weeks after republication.

If we make this change, I don't expect we'll be making it any time soon. This is the big question for the accessibility specification, sort of like adding the html syntax. We want to publicize it and start getting feedback.

**Comment by mattgarrish** [comment]
I forgot to note that this is also why we use different conformance identifier strings. Publications made to conform to 1.0 or 1.1 will remain conforming to those specifications. If we change the baseline for 1.2, only new publications that conform to the new standard will be required to meet the new version/level and they'll have a different identifier.

So, in that sense, we're not breaking conformance, only increasing the requirement to claim conformance to the newest version.

**Comment by GeorgeKerscher** [comment]
Matt wrote: Publications made to conform to 1.0 or 1.1 will remain conforming to those specifications. If we change the baseline for 1.2, only new publications that conform to the new standard will be required to meet the new version/level and they'll have a different identifier.

I think that based on the above, we should move to 1.2

The base would WCAG 2.2 AA.

Would this impact anything with ISO?

**Comment by gregoriopellegrino** [comment]
My question is, has W3C declared WCAG 2.0 obsolete?

WCAG 1.0 is superseded as of May 18, 2021, but 2.0 still appears to be valid.

In my opinion we cannot be more "Catholic than the pope" 😀. Possibly we can discuss this with APA.

Some local laws require specific versions of WCAG, but we work on a technical standard that is applied worldwide, in my opinion we cannot make "political" decisions on the minimum WCAG level.

**Comment by mattgarrish** [comment]
The decision on the task force call today was to not make any change to our current baseline requirement for WCAG 2.0 Level A and recommendations to use the latest version at Level AA.

Since we don't control what producers have to meet for local accessibility laws, we can't make publications that only meet those requirements non-conforming by enforcing higher standards than WCAG 2 itself.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 29 May 2025](https://w3c.github.io/pm-wg/minutes/2025-05-29-a11y.html#1484).

<details><summary><i>View the transcript</i></summary>

<h3>upgrading the floor WCAG specification for EPUB Accessibility 1.1.x from WCAG 2.0 to 2.2.</h3>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2717" rel="noopener noreferrer">w3c/<wbr>epub-specs#2717</a></p>
<p><cite>AvneeshSingh:</cite> should we elevate the floor from WCAG 2.0 level A<br>
<span>… there is no urgency to make this change - WCAG 2 versions do not supersede each other</span></p>
<p><cite>mgarrish:</cite> what about raising just the level?</p>
<p><cite>AvneeshSingh:</cite> could be problematic as some regions still only require level A</p>
<p><cite>AvneeshSingh:</cite> let's continue with the status quo and see if we get any other feedback</p>

<hr /></details>
